src/main/java/net/sf/jabref/gui/maintable/MainTableDataModel.java

```diff
@@ -193,6 +193,8 @@ public void updateGroupFilter() {
 
         private final List<Comparator<BibEntry>> comparators;
 
+
+        @SafeVarargs
```

Comment:
Is this attribute really needed (I guess it complained about unsafe operation, but I do not see any unsafe operation here...) It may go away if you use Arrays.stream() instead of wrapping it in a new list before.